src/maliput_osm/osm/osm_manager.cc

```diff
@@ -46,7 +46,43 @@ void AddToConnections(const Connection& connection, std::vector<osm::Connection>
   }
 }
 
+bool IsPotentialComplexJunction(const std::unordered_map<Lane::Id, maliput::api::LaneEnd::Which>& connections) {
+  return connections.size() >= 2;
+}
+
+void AddLanesToCollectionOfSet(const std::unordered_map<Lane::Id, maliput::api::LaneEnd::Which>& connections,
+                               std::vector<std::unordered_set<std::string>>& sets) {
```

Comment:
Please, use pointer for arguments that are not const.

Reply:
Done.